### PR TITLE
Add ARCH_TYPE to build status file

### PR DESCRIPTION
**Pull request by btoll**
## Summary

Added ARCH_TYPE to file name so they won't overwrite each other.

## Test Plan

n/a

### CODE REVIEW

File: scripts/release/mule/package/deb/package.sh [hunk 1]
```diff
@@ -134,7 +135,7 @@ EOF
 
     pushd "$PKG_DIR"
 
-    STATUSFILE=build_status_${CHANNEL}_${VERSION}
+    STATUSFILE=build_status_${CHANNEL}-${ARCH_TYPE}_${VERSION}
```
Comment (bricerisingalgorand) [inline scripts/release/mule/package/deb/package.sh:138]: Should this be `build_status_${CHANNEL}_${ARCH_TYPE}_${VERSION}`?

Comment (btoll) [reply scripts/release/mule/package/deb/package.sh:138]: I followed the convention of the other files:
```
algorand-devtools_beta_linux-amd64_2.2.0.deb
install_beta_linux-arm_2.2.0.tar.gz
```

Comment (onetechnical) [reply scripts/release/mule/package/deb/package.sh:138]: NO OS_TYPE either?

Comment (onetechnical) [reply scripts/release/mule/package/deb/package.sh:138]: Maybe channel_os-arch_version?

Comment (btoll) [reply scripts/release/mule/package/deb/package.sh:138]: ok, i'll add that

Comment (btoll) [reply scripts/release/mule/package/deb/package.sh]: fixed

**Review by bricerisingalgorand — APPROVED** [review]

**Review by onetechnical — APPROVED** [review]

**Pull request merged**